src/main/java/com/notably/logic/commands/suggestion/SearchSuggestionCommand.java

```diff
@@ -0,0 +1,113 @@
+package com.notably.logic.commands.suggestion;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Queue;
+import java.util.stream.Collectors;
+
+import com.notably.commons.path.AbsolutePath;
+import com.notably.logic.commands.OpenCommand;
+import com.notably.model.Model;
+import com.notably.model.block.BlockTree;
+import com.notably.model.block.BlockTreeItem;
+import com.notably.model.suggestion.SuggestionItem;
+import com.notably.model.suggestion.SuggestionItemImpl;
+
+/**
+ * Represents a suggestion command object to search through all the notes based on keywords.
+ */
+public class SearchSuggestionCommand implements SuggestionCommand {
+    private static final String COMMAND_WORD = "search";
+    private static final String RESPONSE_MESSAGE = "Search through all notes based on keywords";
+    private static final String WHITE_SPACE = "        ";
+
+    private String keywords;
```

Comment:
Nit: Perhaps call this `keyword` instead of `keywords`? At the end of the day it's just a single string